### PR TITLE
feat(SPV-1016): Added webhook functions

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -444,6 +445,37 @@ export class SpvWalletClient {
...
+
+  /**
+   * Admin only: Delete a webhook subscription by the given URL.
+   *
+   * @param url - The URL of the webhook subscription to delete.
+   * @returns A Promise that resolves when the webhook subscription is deleted.
+   */
+  async AdminDeleteWebhook(url: string): Promise<void> {
+    return await this.http.adminRequest(`admin/webhooks/subscriptions`, 'DELETE', { url });
+  }
```
Comment (chris-4chain) [inline src/client.ts:477]: Please add basic example (to /examples) how to subscribe, getlist, and unsubscribe a webhook.
```javascript
//pseudo code
webhookURL = "http://example.com"
client.AdminSubscribeWebhook(webhookURL)
list = AdminGetWebhooks()
print(list)
client.AdminDeleteWebhook(webhookURL)
```

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: src/client.ts (hunk 1 above)
Comment (Nazarii-4chain) [reply src/client.ts:477]: Added examples

File: examples/admin-get-webhooks.ts
Comment (chris-4chain) [inline examples/admin-get-webhooks.ts:1]: Maybe you misunderstand me
> Please add basic example

I meant one example instead of three.

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: examples/admin-get-webhooks.ts
Comment (Nazarii-4chain) [reply examples/admin-get-webhooks.ts:1]: Updated a file

File: examples/admin-webhooks.ts
Comment (chris-4chain) [inline examples/admin-webhooks.ts:1]: Each example should be described in documentation in [examples/README.md](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/examples/README.md) and added to [examples/package.json](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/examples/package.json).

Do it similarly to already existed examples.

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: examples/admin-webhooks.ts
Comment (Nazarii-4chain) [reply examples/admin-webhooks.ts:1]: Updated

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**